### PR TITLE
Update country.interface.ts

### PR DESCRIPTION
**Pull request by alamuk**

### CODE REVIEW

File: src/app/orders/models/country.interface.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ export interface Country{
     countryId: string,
     countryC3: string,
     countryN3: string,
-    countryName: string
-}
\ No newline at end of file
+    countryName: string,
+    countryDivision: String,
```
Comment (oscarlagatta) [inline src/app/orders/models/country.interface.ts]: this is wrong, String should be all lowercase

**Comment by oscarlagatta** [comment]
Hi Alan, pls when possible correct the error and create a new pull request, thank you

**Comment by alamuk** [comment]
Hi Oscar ,
Thank you very much for correcting me.

Sorry it was a test to send the pull request.

I will send the new request soon within 1 hour.






Sent from my iPhone

On 28 Mar 2018, at 4:53 pm, Oscar Lagatta <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Alan, pls when possible correct the error and create a new pull request, thank you

**Comment by oscarlagatta** [comment]
Hey Alam,

take it easy, not to worry whenever possible, well done creating the pull
requests, amazing job.

Oscar

**Pull request closed without merging**